src/metabase/email/messages.clj

```diff
@@ -226,3 +226,121 @@
                        (vec (cons :div (for [result results]
                                          (render/render-pulse-section timezone result)))))]
     (render-message-body "metabase/email/pulse" (pulse-context body pulse) (seq @images))))
+
+(defn pulse->alert-condition-kwd
+  "Given an `ALERT` return a keyword representing what kind of goal needs to be met."
+  [{:keys [alert_above_goal alert_condition card creator] :as alert}]
+  (if (= "goal" alert_condition)
+    (if (true? alert_above_goal)
+      :meets
+      :below)
+    :rows))
```

Comment:
Nitpicky but I think nested if statements get a little tricky to read in Clojure. One `cond` I think is easier to read:

``clojure
(defn pulse->alert-condition-kwd
  "Given an `ALERT` return a keyword representing what kind of goal needs to be met."
  [{:keys [alert_above_goal alert_condition card creator] :as alert}]
  (cond
    (= "goal" alert_condition) :rows
    (true? alert_above_goal)   :meets
    :else                      :below))
```

Also what's the reason behind checking if `alert_above_goal` is `true`? Is it not always boolean? If it's set to some string or numeric value do we want to treat it as false? That should be noted somewhere IMO otherwise it just looks like the code was written wrong accidentally